src/check.h.in

```diff
@@ -987,6 +1000,33 @@ CK_DLL_EXP void CK_EXPORT srunner_run(SRunner * sr, const char *sname,
 
 
 /**
+ * Run a specific suite or test case or testcases with specific tags
```

Comment:
You will need to update the comments on the srunner_run_all and srunner_run functions above to include the following:
```
 * Note that if the CK_RUN_CASE, CK_RUN_SUITE, CK_INCLUDE_TAGS,
  * and/or CK_EXCLUDE_TAGS environment variables are defined, then only
  * the selected suite or test cases will run.
```
namely:
```
 /**
  * Runs a suite runner and all contained suite, printing results to
  * stdout as specified by the print_mode.
  *
  * In addition to running all suites, if the suite runner has been
  * configured to output to a log, that is also performed.
  *
  * Note that if the CK_RUN_CASE, CK_RUN_SUITE, CK_INCLUDE_TAGS,
  * and/or CK_EXCLUDE_TAGS environment variables are defined, then only
  * the selected suite or test cases will run.
  *
  * @param sr suite runner to run all suites from
  * @param print_mode the verbosity in which to report results to stdout
  *
  * @since 0.6.0
  */
 CK_DLL_EXP void CK_EXPORT srunner_run_all(SRunner * sr,
                                           enum print_output print_mode);

 /**
  * Run a specific suite or test case from a suite runner, printing results
  * to stdout as specified by the print_mode.
  *
  * In addition to running any applicable suites or test cases, if the
  * suite runner has been configured to output to a log, that is also
  * performed.
  *
  * Note that if the CK_RUN_CASE, CK_RUN_SUITE, CK_INCLUDE_TAGS,
  * and/or CK_EXCLUDE_TAGS environment variables are defined, then only
  * the selected suite or test cases will run.
  *
  * @param sr suite runner where the given suite or test case must be
  * @param sname suite name to run. A NULL means "any suite".
  * @param tcname test case name to run. A NULL means "any test case"
  * @param print_mode the verbosity in which to report results to stdout
  *
  * @since 0.9.9
  */
 CK_DLL_EXP void CK_EXPORT srunner_run(SRunner * sr, const char *sname,
                                       const char *tcname,
                                       enum print_output print_mode);
```